sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/SessionReceiverManager.cs

```diff
@@ -295,10 +295,7 @@ public override async Task ReceiveAndProcessMessagesAsync(CancellationToken proc
                 }
             }
             catch (Exception ex)
-            when (ex is not TaskCanceledException ||
-            // If the user manually throws a TCE, then we should log it.
-            (!_sessionCancellationSource.IsCancellationRequested &&
```

Comment:
This was not needed because TCE thrown by user handler were already being logged within ProcessOneMessage